src/wazuh_db/wdb.c

```diff
@@ -1145,6 +1146,51 @@ cJSON * wdb_exec_stmt(sqlite3_stmt * stmt) {
     return result;
 }
 
+cJSON * wdb_exec_stmt_single_array(sqlite3_stmt * stmt) {
```

Comment:
I think a name like wdb_exec_stmt_single_column will be more representative.

Reply:
You are right!
Fixed!